README.md

```diff
@@ -275,6 +275,7 @@ See also the following lists dedicated to specifics sub-topics:
 * [pwndocker][458]: docker environment for pwn in ctf.
 * [pwninit][457]: automate starting binary exploit challenges.
 * [pwntools][71]: framework and exploit development library.
+* [ronin-exploits][663] - A Ruby micro-framework for writing and running exploits and payloads.
```

Comment:
Thanks for the contribution. Could you please update the PR and substitute the dash `[663] -` with a colon `[663]:`?

Reply:
My mistake. Should be fixed now.